src/content/app-platform/overview/index.md

```diff
@@ -31,6 +31,7 @@ you to browse, install and manage the configurations of apps (such as Prometheus
 from a single place; the management cluster.
 
 Using this platform, we are providing a collection of curated _Apps_. These _Apps_ are grouped into _App Catalogs_, which are browsable through our web interface.
+We also use app platform to install the apps that are pre-installed in your cluster (such as CoreDNS).
```

Comment:
Adding this to make it clear that app platform is used even if the customer is not using managed apps.
See https://github.com/giantswarm/giantswarm/issues/16903

@Oshratn PTAL 🙏